misc/services/acrn_manager/acrn_vm_ops.c

```diff
@@ -115,7 +115,8 @@ static void _scan_alive_vm(void)
 
 	ret = check_dir(ACRN_DM_SOCK_PATH, CHK_ONLY);
 	if (ret) {
-		printf("%s: Failed to check directory %s, err: %d\n", __func__, ACRN_DM_SOCK_PATH, ret);
+		printf("acrnd may not be launched yet. %s: Failed to check directory %s, err: %d\n",
```

Comment:
```suggestion
		printf("%s: Failed to check directory %s, err: %d\nMake sure the acrnd daemon is running on your system.\n",
```

Reply:
> 

Even better would be to add how to check that acrnd daemon is running...

Reply:
True... the canonical way of doing this would be: `systemctl status acrnd`, **but** I'm not sure the whole thing is working as expected, I am wondering about these couple of things in particular:
* Is the `acrnd.service` enabled by default?
* Is the `After=weston.service` not a problem in Ubuntu?

Reply:
Hi Geoffroy and David, thanks for your review and comments.
(1)acrnd.service needs to be enabled by user manually.
(2) weston is the reference implementation of a Wayland server. A Wayland server is a display server, a window manager, and a compositor all in one. Weston service is optional for ACRN, so I will remove this configuration in acrnd service in another patch, thanks!

Reply:
> (1)acrnd.service needs to be enabled by user manually.

That's what I thought, thanks for confirming this. We need to update our documentation as it's not mentionned anywhere, it's actually slightly worse than that since we let the user believe it is enabled by default:
```
A systemd service file (acrnd.service) is installed by default that will start the acrnd daemon when the Service VM (Linux-based) comes up. You can restart/stop acrnd service using systemctl
```

Reply:
Hi Geoffroy, get it, I have updated document in this PR, please help double check it, thanks!